docs/concepts/events-vs-filters.rst

```diff
@@ -0,0 +1,73 @@
+Events vs Filters
+=================
+
+Open edX Events and Filters are two types of hooks that allow developers to extend the functionality of the Open edX platform. They are defined in the `openedx-events`_ and `openedx-filters`_ libraries respectively.
+
+.. _openedx-events: https://github.com/openedx/openedx-events
+.. _openedx-filters: https://github.com/openedx/openedx-filters
+
+Events
+------
+
+Events are Open edX-specific Django signals sent in specific places in the Open edX platform. Signal handlers can then be hooked into these signals to perform additional processing based on the event data.
+
+For a more detailed explanation of Open edX Events, see the rest of the Open edX Events documentation.
+
+Filters
+-------
+
+Filters are functions that can modify the application's behavior by altering input data or halting execution based on specific conditions. They allow developers to implement application flow control based on their business logic or requirements without directly modifying the application code.
```

Comment:
Here we should use the official definition for both, so we don't maintain definitions in multiple places.